.github/ISSUE_TEMPLATE/epic.yml

```diff
@@ -1,104 +1,69 @@
-name: Epic
-description: Create an epic
+name: Epic (User Impacting)
+description: An epic is a grouping of related issues which MUST be delivered together. Epics are as small in scope as reasonably possible, and are "finite" in scope. Please do not use this template for: categorising related work (please instead use labels), investigate work where the solutions is not well defined (e.g. investgiations + spikes), or for technical debt work.
 title: "Epic: "
 labels: ["type: epic"]
 body:
-- type: markdown
-  attributes:
-    value: Before raising an epic, please search for existing epics[[1](https://github.com/gitpod-io/gitpod/issues?q=is%3Aopen+is%3Aissue+label%3A%22type%3A+epic%22)][[2](https://github.com/gitpod-io/gitpod/issues?q=is%3Aopen+is%3Aissue+%22Epic%3A+%22)] to avoid creating duplicates. Read more about [Epics](https://www.notion.so/gitpod/Development-Process-2-0-6681854173ab4d2f92880f9f3d85cab5#321619f5a4bd4391be83c66feb2cdb49) (internal) in **Development Process**.
```

Comment:
I believe this "don't duplicate" can be implied, I'm not sure it adds much value, only select Gitpodders are really creating or concerned with epics. Also, I think we should remove the template from the RFC, and have this template as the single source of truth, making a link back redundant.